src/net.h

```diff
@@ -289,6 +289,8 @@ class CConnman
 
     unsigned int GetReceiveFloodSize() const;
 
+    int GetMaxConnections();
+    int GetMaxAllowedConnections();
```

Comment:
Nitpick: Looks like these methods are in reverse order in the .cpp file, so I would suggest reversing them here.